resources/qml/UserProfile.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.3
+
+import im.nheko 1.0
+
+ApplicationWindow{
+    property var user_data
+    property var colors: currentActivePalette
+
+    id:userProfileDialog
+    height: 500
+    width: 500
+    modality:Qt.WindowModal
+    Layout.alignment: Qt.AlignHCenter
+    palette: colors
+
+    onAfterRendering: {
```

Comment:
This looks weird, you should be binding those directly not after renderig

Reply:
Yeah I'll Look into it